Upgrade to React 17?

The `peerDependency` in this package means that it can't be installed in a project that uses React v17 and above. I'm not sure if there are compatibility issues with 17 or not. Perhaps it's just a versioning change that's needed?

Here's what I saw when trying to install in my project with a react dependency of `"react": "^17.0.2"`:

> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR! 
> npm ERR! While resolving: lichfield-live@0.1.0
> npm ERR! Found: react@17.0.2
> npm ERR! node_modules/react
> npm ERR!   react@"^17.0.2" from the root project
> npm ERR! 
> npm ERR! Could not resolve dependency:
> npm ERR! peer react@"^16.8" from react-wordpress-comments@0.0.2
> npm ERR! node_modules/react-wordpress-comments
> npm ERR!   react-wordpress-comments@"*" from the root project
> npm ERR! 
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
     "test:watch": "nwb test-react --server"
   },
   "peerDependencies": {
-    "react": "^16.8"
+    "react": "^16.8.0 || ^17.0.0"
   },
   "devDependencies": {
     "eslint": "^7.22.0",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-wordpress-comments",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "A react component for headless WordPress that fetches and publishes comments via REST API",
   "author": "Karen Sarkisyan <karen.sarkisyan01@gmail.com> (https://github.com/karen-sarkisyan)",
   "homepage": "https://github.com/karen-sarkisyan/react-wordpress-comments#readme",

--- a/src/createComment.js
+++ b/src/createComment.js
@@ -15,7 +15,7 @@ function CreateComment(props) {
     }
     // if provided, add JWT auth header
     if (props.user) {
-      reqHeaders.Authorization = `Bearer: ${props.user.jwt}`
+      reqHeaders.Authorization = `Bearer ${props.user.jwt}`
     }
 
     let payload = {

